spark-doris-connector/src/main/java/org/apache/doris/spark/serialization/RowBatch.java

```diff
@@ -369,28 +374,23 @@ public void convertArrowToRowBatch() throws DorisException {
                                     continue;
                                 }
                                 String value = new String(varCharVector.get(rowIndex), StandardCharsets.UTF_8);
-                                addValueToRow(rowIndex, value);
+                                 value = completeMilliseconds(value);
+                                LocalDateTime parse = LocalDateTime.parse(value, dateTimeV2Formatter);
```

Comment:
Is there a problem with using dateTimeV2Formatter for datetime?

Reply:
> Is there a problem with using dateTimeV2Formatter for datetime?

I tested it and referred to the flink connector.